app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_storage = {}
+
+    def wrapper(*args) -> None:
+        if args in cache_storage:
```

Comment:
Using mutable types (like lists) as dictionary keys can lead to errors because they are not hashable. Ensure that the arguments used as keys in `cache_storage` are immutable (e.g., tuples).